api/src/services/monitoring.js

```diff
@@ -86,45 +100,89 @@ const getCouchVersion = () => {
 };
 
 const defaultNumber = x => typeof x === 'number' ? x : -1;
+const defaultBoolean = x => typeof x === 'boolean' ? x : false;
```

Comment:
I am conflicted on this.  On one hand, `false` seems like a bad default value because it has semantic meaning (e.g. there is no way to tell if Couch returned no data from `compact_running` or if there is actually no compaction running).  

On the other hand, returning an empty string or `undefined` for a boolean JSON field seems a bit unexpected.... :thinking: